tests/end_to_end/test_docker_monitoring/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Test case configuration
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Uninstall old versions of Docker (CentOS)
+      become: True
+      shell: >
+        yum -y remove docker
+        docker-client
+        docker-client-latest
+        docker-common
+        docker-latest
+        docker-latest-logrotate
+        docker-logrotate
+        docker-engine
+        docker-ce
+        docker-ce-cli
+        containerd.io
+        docker-compose-plugin
+      when: ansible_facts['distribution'] == "CentOS"
+
+    - name: Uninstall old versions of Docker (Ubuntu)
+      become: True
+      shell: >
+        apt-get -y remove docker
+        docker-engine
+        docker.io
+        containerd
+        runc && apt-get -y purge docker-ce
+        docker-ce-cli
+        containerd.io
+        docker-compose-plugin
+      when: ansible_facts['distribution'] == "Ubuntu"
+
+    - name: Install Docker using the convenience script
+      become: True
+      shell: curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh
+
+    - name: Start Docker with systemd
+      become: True
+      shell: systemctl stop docker && systemctl stop docker.socket && systemctl start docker
+
+    - name: Configure the docker-listener module
+      become: True
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        insertbefore: "^</ossec_config>"
+        block: |
+          <wodle name="docker-listener">
+            <interval>10m</interval>
+            <attempts>5</attempts>
+            <run_on_start>yes</run_on_start>
+            <disabled>no</disabled>
+          </wodle>
+        marker: "<!-- {mark} ANSIBLE MANAGED BLOCK -->"
+
+    - name: Restart wazuh-manager
+      become: True
+      shell: systemctl restart wazuh-manager
+
+    - name: Unzip Wazuh installation files
+      become: True
+      unarchive:
+        src: /root/wazuh-install-files.tar
+        dest: /root
+        remote_src: yes
+
+    - name: Get credentials file
+      become: True
+      fetch:
+        src: /root/wazuh-install-files/passwords.wazuh
+        dest: /tmp/
+        flat: yes
```

Comment:
We are going to delete these tasks and the credentials will be in the inventory

Reply:
We need to remove these tasks from the playbook

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/2948/commits/be397a6c1614df98c13b422110389a1d656d6eb8